draft-ietf-oauth-device-flow.xml

```diff
@@ -637,6 +641,12 @@
     <spanx style="verb">slow_down</spanx> which are processed as
     described above.
   </t>
+  <t>
+  Clients on encountering a connection timeout MUST unilaterally reduce
```

Comment:
This would read more naturally as "On encountering a connection timeout, client MUST"...

Reply:
Done